test/addons/rook-pool/storage-class.yaml

```diff
@@ -7,7 +7,7 @@ kind: StorageClass
 metadata:
     name: rook-ceph-block
     labels:
-        ramendr.openshift.io/storageid: rook-ceph-storage-id-on-$cluster
+        ramendr.openshift.io/storageid: rook-ceph-storage-id-$cluster-1
```

Comment:
The `storage-id` part is not needed, the label key is `ramendr.openshift.io/storageid`, so `rook-ceph-$cluster-1`?

Also squashing the second commit will be helpful.